amd_openvx_extensions/amd_custom/include/vx_amd_custom.h

```diff
@@ -58,7 +54,7 @@ typedef struct _vx_amd_custom_params_t
  * \param [in] graph The handle to the graph.
  * \param [in] inputs The input tensor data.
  * \param [in] function custom funtion enum.
- * \param [in] array for user specified custom_parameters.
+ * \param [in] custom_parameters for user specified custom_parameters.
```

Comment:
@fiona-gladwin Please say custom_parameters <tt> vx_array </tt> for user specified... Needs to say what kind of input. Check on all files if any missing

Reply:
Done